RxExample/RxExample/Examples/TableViewWithEditingCommands/TableViewWithEditingCommandsViewController.swift

```diff
@@ -104,6 +104,10 @@ class TableViewWithEditingCommandsViewController: ViewController, UITableViewDel
                     SectionModel(model: "Normal Users", items: $0.users)
                 ]
             }
+            .catchError { (error) in
```

Comment:
Why not just adding `catchError` just after `getExampleUserResultSet()`? I'm not sure but it's enough just adding:

```diff
  let loadFavoriteUsers = RandomUserAPI.sharedAPI
    .getExampleUserResultSet()
+   .catchErrorJustReturn(TableViewEditingCommand.setUsers([]))
    .map(TableViewEditingCommand.setUsers)
```